### ISSUE TITLE
[Documentation] Add Contribution Guide to README 

### ISSUE THREAD

**Issue by MalHus** [WISE-UofT/wise-main#7]
Add a section to the `README` that outlines the steps for contributing to the repository. This will help new contributors understand how to fork, clone, make changes and open a pull request. 

#### Pre-requisites
Install [Visual Studio Code](https://code.visualstudio.com/) and the [Live Server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer) extension.

#### Create and Develop on a Fork of `wise-main`
1. Fork the `wise-main` repository
2. Go to the forked repository and clone it to your local machine
4. Create a new branch with your changes and complete your changes 
6. Push your branch to your **forked** repository

#### Opening a Pull Request to the Original `wise-main` Repository
1. Go to your **forked** repository on Github
2. With the new changes pushed, go to the "Pull Requests" tab and click "New Pull Request"
3. In the `base repository` dropdown, select `WISE-UofT/wise-main`. In the `base` dropdown selection the branch you want to merge into, usually `main`
4. In the `head respository` dropdown, select your forked repository name. In the `compare` dropdown, select the branch you just pushed 
5. Select `Allow edits by maintainers`
<img width="1077" alt="image" src="https://github.com/user-attachments/assets/29ff0a38-1368-4724-8fd6-13d2ea40552c">
6. Add a title, description and click "Create Pull Request"

#### Wait for Approval
The VP of marketing can now see this PR in the original `wise-main` repo under the "Pull Requests" tab.

**Comment by helenaglow** [comment]
Wait can you not just clone the repo, why do you have to fork? Maybe I am just being silly

**Comment by MalHus** [comment]
> Wait can you not just clone the repo, why do you have to fork? Maybe I am just being silly

All good, this was new experiance for me as well! So, I looked through to see how Gaurika (prev webmaster) did it and she also was instructed to make a fork of `wise-main`. Since I dont have write access to the repo (I made a test push and I was **not** allowed), I used a [fork and pull request workflow.](https://docs.github.com/en/get-started/exploring-projects-on-github/contributing-to-a-project). Its more steps on my end but I thought since Gaurika did it this way and you would not need to give me any permissions, I went ahead with it. However, if you have the ability to give me write access, it would def be easier on my end!
